### PR TITLE
Drop admin section in devhub; move to django admin page.

### LINKED ISSUE #7657: Prevent changing the extension type from Admin Settings in devhub
STR:
1. Load AMO dev and from Dev hub go to Manage My Submissions.
2. Click on Edit information for a submitted add-on.
3. From Admin Settings change the add-on type from Extension to Theme.
4. Go back to Manage my submissions and try to access My Themes tab.

Expected result:
My Themes tab is displayed.

Actual result:
Oops! We had an error. The content of the theme tab is not visible.

Notes:
This issue is reproducible in AMO dev with FF58 (Win10).

oops

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #7657

### CODE REVIEW

File: src/olympia/addons/admin.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ class Media:
         ('Truthiness', {
             'fields': ('disabled_by_user', 'view_source', 'requires_payment',
                        'public_stats', 'is_experimental',
-                       'external_software', 'dev_agreement'),
+                       'external_software', 'dev_agreement', 'reputation'),
```
[the remark this answers is not in the document]
Comment (diox) [reply src/olympia/addons/admin.py]: I'd just rename the fieldset to "Flags" tbh

[the remark this answers is not in the document]
Comment (diox) [reply src/olympia/addons/admin.py]: Or "Random properties we have no idea where to put" :p

[the remark this answers is not in the document]
Comment (eviljeff) [reply src/olympia/addons/admin.py]: fwiw, pretty sure `dev_agreement` is unused.  `read_dev_agreement` is a `UserProfile` property instead.

[the remark this answers is not in the document]
Comment (eviljeff) [reply src/olympia/addons/admin.py]: https://github.com/mozilla/addons-server/pull/8623/commits/4e0d325a3c3a5aca7c2f2fa085268c9fe6bab882

[the remark this answers is not in the document]
Comment (diox) [reply src/olympia/addons/admin.py]: Eh, right, never noticed that. File an issue to remove it?

**Pull request merged**